### PR TITLE
add collageplus v0.3.3

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/ed-lea/jquery-collagePlus
**Watch 75
  Star 1,094
 Fork 185**
@Amomo could u help me check this PR for #6776 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Looks like we should add `extras` and `css` folder.

**Comment by maruilian11** [comment]
@Amomo i have add `css` and `extras`, could you help me check this PR? thanks.

File: ajax/libs/collageplus/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ed-lea/jquery-collagePlus.git",
+    "basePath": "",
+    "files": [
+      "jquery.collagePlus*.js",
+      "css/transitions*.css",
+      "extras/jquery.collageCaption*.js",
+      "extras/jquery.removeWhitespace*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/collageplus/package.json]: What about `"extras/jquery*"

**Comment by maruilian11** [comment]
@Amomo i have modified the filemap to `extras/jquery*` could you help me check this PR? thanks!

**Comment by PeterDaveHello** [comment]
Failed ...

**Comment by maruilian11** [comment]
@Amomo i have add `css` and `extras`, could you help me check this PR? thanks.

**Pull request merged**